recipes/mppp/all/conanfile.py

```diff
@@ -58,56 +64,57 @@ def requirements(self):
         if self.options.with_mpc == True:
             self.requires("mpc/1.2.0")
         if self.options.with_boost == True:
-            self.requires("boost/1.79.0")
+            self.requires("boost/1.80.0")
 
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, 11)
+            check_min_cppstd(self, self._minimum_cpp_standard)
         if self.options.with_arb:
-            raise ConanInvalidConfiguration("{}/{} doesn't supported arb (yet)".format(self.name, self.version))
+            raise ConanInvalidConfiguration(f"{self.ref} doesn't supported arb (yet)")
```

Comment:
Seems a little odd to have options that are not supported by the project?

The message might need some improvement... is the version too high in CCI? IS it not offered here yet?

```suggestion
            raise ConanInvalidConfiguration(f"{self.ref} doesn't support arb (yet)")
```